docs/hub/model-card-annotated.md

```diff
@@ -0,0 +1,305 @@
+
+**Section Overview:**  Provide the model name and a 1-2 sentence summary of what the model is. 
+
+{{ model_id | default("Model ID", true) }}
+{{ model_summary | default("", true) }}
+
+# Table of Contents
+
+**Section Overview:** Provide this with links to each section, to enable people to easily jump around/use the file in other locations with the preserved TOC/print out the content/etc.
+
+# Model Details
+
+**Section Overview:** This section provides basic information about what the model is, its current status, and where it came from. It should be useful for anyone who wants to reference the model.
+
+
+## Model Description
+
+
+{{ model_description | default("", true) }}
+
+_Provide basic details about the model. This includes the architecture, version, if it was introduced in a paper, if an original implementation is available, and the creators. Any copyright should be attributed here. General information about training procedures, parameters, and important disclaimers can also be mentioned in this section._
+
+
+* **Developed by:** {{ developers | default("[More Information Needed]", true)}}
+
+_List (and ideally link to) the people who built the model._
+
+
+
+* **Shared by [optional]:** {{ shared_by | default("[More Information Needed]", true)}}
+
+_List (and ideally link to) the people/organization making the model available online._
+
+
+
+* **Model type:** {{ model_type | default("[More Information Needed]", true)}}
+
+_You can name the “type” as:_
+
+_1. Supervision/Learning Method_
+
+_2. Machine Learning Type_
+
+_3. Modality_
+
+* **Language(s)** [NLP]: {{ language | default("[More Information Needed]", true)}}
+
+_Use this field when the system is a natural language processing system._
+
+
+
+* **License:** {{ license | default("[More Information Needed]", true)}}
+
+_Name, and link to, the license being used._
+
+
+
+* **Finetuned From Model [optional]:** {{ finetuned_from | default("[More Information Needed]", true)}}
+
+_If this model has another model as its base, link to that model here._
+
+
+## Model Sources [optional]
+
+* **Repository:** {{ repo | default("[More Information Needed]", true)}}
+* **Paper [optional]:** {{ paper | default("[More Information Needed]", true)}}
+* **Demo [optional]:** {{ demo | default("[More Information Needed]", true)}}
+
+_Provide sources for the user to directly see the model and its details. Additional kinds of resources – training logs, lessons learned, etc. – belong in the [More Information](#more-information) section. If you include one thing for this section, link to the repository._
+
+# Uses
+**Section Overview:** This section addresses questions around how the model is intended to be used, discusses the foreseeable users of the model (including those affected by the model), and describes uses that are considered out of scope or misuse of the model.  Note this section is not intended to include the license usage details. For that, link directly to the license.
+
+
+## Direct Use
```

Comment:
The difference between Direct Use and Downstream use are not very clear to me. 

On one, we would want to explain how to use the model for fine-tuning.
On the other, we would like to show how to use the model without fine-tuning (e.g. for running inference in an app).

Is that right? The descriptions seem to mix a bit both.

Reply:
Hmm. One says "without fine-tuning", then the other says "when fine-tuned". Do you have a wording suggestion to make it less confusing?

Reply:
(Explaining how to fine-tune the model isn't what's happening here: this section is for how the model is used in different applied contexts.)

Reply:
Added "in different applied contexts". Does that help at all?

Reply:
Yep this is good. I think this is good!